packages/cli-plugin-metro/src/tools/loadMetroConfig.ts

```diff
@@ -14,9 +14,14 @@ const INTERNAL_CALLSITES_REGEX = new RegExp(
     '/Libraries/YellowBox/.+\\.js$',
     '/Libraries/LogBox/.+\\.js$',
     '/Libraries/Core/Timers/.+\\.js$',
+    '/Libraries/WebSocket/.+\\.js',
+    '/Libraries/vendor/.+\\.js',
     '/node_modules/react-devtools-core/.+\\.js$',
     '/node_modules/react-refresh/.+\\.js$',
     '/node_modules/scheduler/.+\\.js$',
+    '/node_modules/event-target-shim/.+\\.js$',
+    '/metro-runtime/.+\\.js$',
+    '\\[native code\\]',
```

Comment:
```suggestion
    '^\\[native code\\]$',
```

I suspect this string may be engine-specific - have you tested this on both JSC and Hermes?

Also, bikeshedding: might be useful to include native code as intermediate frames, though I agree they're less useful at the top since we have no JS code frame to show there.